CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ Currently the versioning policy of this project follows [Semantic Versioning v2.
 * Function `mutableSignature()` improved and factored out from the `MutableStaticFields` detector
 
 ### Added
+* New detector `FindBadEndOfStreamCheck` for new bug type `EOS_BAD_END_OF_STREAM_CHECK`. This bug is reported whenever the return value of java.io.FileInputStream.read() or java.io.FileReader.read() is first converted to byte/int and only thereafter checked against -1. (See [SEI CERT rule FIO08-J](https://wiki.sei.cmu.edu/confluence/display/java/FIO08-J.+Distinguish+between+characters+or+bytes+read+from+a+stream+and+-1))
```

Comment:
Could you move this line to the bottom of `## Unreleased`? Then I'll merge and will be released as a part of the next release, probably `4.4.0`.

Reply:
To ease the merge process I tried to create a branch to merge both PRs in batch: https://github.com/spotbugs/spotbugs/compare/PR-from-baloghadamsoftware 
However, I used squashed commit so it loses author Infos in the commit. I will try another way without squash, but if you're OK to update your PRs, please update yours one by one.

Reply:
also created https://github.com/spotbugs/spotbugs/compare/PR-from-balonghadam2 with no squash commit, but its commit history seems non intuitive... I'll wait @baloghadamsoftware's conflict resolve.